ql/src/java/org/apache/hadoop/hive/ql/ddl/table/storage/skewed/AlterTableSetSkewedLocationDesc.java

```diff
@@ -57,6 +57,6 @@ public AlterTableSetSkewedLocationDesc(TableName tableName, Map<String, String>
 
   @Override
   public boolean mayNeedWriteId() {
-    return false;
+    return true;
```

Comment:
AFAIK we do not support skewed ACID tables

Reply:
Flipping back to false